src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -0,0 +1,325 @@
+                      selected={role.name === value}
+                      key={role.name}
+                      value={role.name}
+                    />
+                  ))}
+                </Select>
+              )}
+            />
+          </FormGroup>
+        </FormAccess>
+      </FormPanel>
+      <FormPanel className="kc-permissions" title={t("permissions")}>
+        <FormAccess isHorizontal role="manage-clients">
+          <FormGroup
+            label={t("applyToResourceType")}
+            fieldId="applyToResourceType"
+            labelIcon={
+              <HelpItem
+                helpText="clients-help:applyToResourceType"
+                fieldLabelId="clients:applyToResourceType"
+              />
+            }
+          >
+            <Controller
+              name="applyToResource"
+              defaultValue=""
+              control={control}
+              render={({ onChange, value }) => (
+                <Switch
+                  id="applyToResource-switch"
+                  label={t("common:on")}
+                  labelOff={t("common:off")}
+                  isChecked={value === "true"}
+                  onChange={(value) => onChange("" + value)}
+                />
+              )}
+            />
+          </FormGroup>
+          <FormGroup
+            label={t("resourcesAndAuthScopes")}
+            id="test"
+            labelIcon={
+              <HelpItem
+                helpText={t("clients-help:contextualAttributes")}
+                fieldLabelId={`contextualAttributes`}
+              />
+            }
+            helperTextInvalid={t("common:required")}
+            fieldId={name!}
+          >
+            <AttributeInput
+              selectableValues={resources.map((item) => item.name!)}
+              resources={resources}
+              isKeySelectable
+              name="resources"
+            />
+          </FormGroup>
+          <ExpandableSection
+            toggleText={t("contextualInfo")}
+            onToggle={() => setIsExpanded(!isExpanded)}
+            isExpanded={isExpanded}
+          >
+            <FormGroup
+              label={t("contextualAttributes")}
+              id="test"
+              labelIcon={
+                <HelpItem
+                  helpText={t("clients-help:contextualAttributes")}
+                  fieldLabelId={`contextualAttributes`}
+                />
+              }
+              helperTextInvalid={t("common:required")}
+              fieldId={name!}
+            >
+              <AttributeInput
+                selectableValues={defaultContextAttributes.map(
+                  (item) => item.name
+                )}
+                isKeySelectable
+                name="context"
+              />
+            </FormGroup>
+          </ExpandableSection>
+        </FormAccess>
+        <ActionGroup>
+          <Button data-testid="authorization-eval" onClick={evaluate}>
+            {t("evaluate")}
+          </Button>
```

Comment:
instead of calling this function directly we can make it a form submit
```suggestion
          <Button data-testid="authorization-eval" type="submit">
            {t("evaluate")}
          </Button>
```